core/kernel/classes/class.Property.php

```diff
@@ -166,6 +166,13 @@ public function getRelatedClass(): ?core_kernel_classes_Class
         }
     }
 
+    public function isStatistical(): bool
+    {
+        $value = $this->getOnePropertyValue(new core_kernel_classes_Property(GenerisRdf::PROPERTY_IS_STATISTICAL));
```

Comment:
I think, you can use `$this->getProperty()` instead of initialization.
```suggestion
        $value = $this->getOnePropertyValue($this->getProperty(GenerisRdf::PROPERTY_IS_STATISTICAL));
```

Reply:
Done, thanks! :)